### ISSUE TITLE
Question about Fine-tuning Configuration on NYU Dataset

### ISSUE THREAD

**Issue by dxw2000** [lpiccinelli-eth/UniDepth#137]
In Table III of the paper, the performance on the NYU validation set is evaluated. Therefore, I assume that your method was fine-tuned on the NYU dataset. Could you please share the fine-tuning configuration used in your experiments? Specifically, did you fine-tune only certain layers, or was a full-parameter fine-tuning performed? I would greatly appreciate any details you could provide about the fine-tuning setup described in the paper.

**Comment by Cats-eat-peppercorns** [comment]
@dxw2000 朋友，请问你微调后nyuv2验证结果中d1能达到多少？d2能达到多少？silog能达到多少呢？论文里描述d1是98.8%，而我只能达到97.3%。请问微调过程是直接用这个仓库里指明的预训练模型进行呢？还是怎样？你微调了多少步呢？学习率设置是多少呢？下面是我的配置，向你请教一下。`{
    "_comment": "NYUv2-only ViT-L 微调 config — 用于复现论文 NYUv2 指标",
    "_description": "基于原始 UniDepth V2 config_v2_vitl14.json，适配 NYUv2-only 训练。关键调整：lr 降低、iterations 减少、无 warmup、early stopping 启用、推理时使用 GT 相机内参。数据集使用标准 nyuv2.hdf5 (depth_scale=1000, uint16 mm)。",

    "generic": {
        "seed": 13,
        "deterministic": true
    },

    "training": {
        "n_iters": 15000,
        "batch_size": 8,
        "validation_interval": 500,
        "nsteps_accumulation_gradient": 2,
        "use_checkpoint": false,
        "find_unused_parameters": false,

        "lr": 5e-5,
        "lr_final": 1e-7,
        "lr_warmup": 1.0,
        "cycle_beta": false,

        "pretrained": "/home/xzy/weights/pytorch_model_l.bin",

        "wd": 0.1,
        "wd_final": 0.1,
        "warmup_iters": 0,
        "ld": 1.0,

        "drop_path": 0.0,
        "ema": true,
        "f16": true,
        "clipping": 1.0,

        "losses": {
            "depth": {
                "name": "SILog",
                "weight": 1.0,
                "output_fn": "sqrt",
                "input_fn": "log",
                "dims": [-2, -1],
                "integrated": 0.15
            },
            "invariance": {
                "name": "SelfDistill",
                "weight": 0.1,
                "output_fn": "sqrt"
            },
            "camera": {
                "name": "Regression",
                "weight": 0.25,
                "gamma": 1.0,
                "alpha": 1.0,
                "fn": "l2",
                "output_fn": "sqrt",
                "input_fn": "linear"
            },
            "ssi": {
                "name": "EdgeGuidedLocalSSI",
                "weight": 1.0,
                "output_fn": "sqrt",
                "input_fn": "log1i",
                "use_global": true,
                "min_samples": 6
            },
            "confidence": {
                "name": "Confidence",
                "weight": 0.1,
                "gamma": 1.0,
                "alpha": 1.0,
                "fn": "l1",
                "output_fn": "sqrt",
                "input_fn": "linear"
            }
        },

        "ema_update_after_step": 4000,
        "ema_tau": 5000,

        "early_stopping": {
            "enable": true,
            "patience": 5000,
            "metric": "d1",
            "mode": "max"
        }
    },

    "data": {
        "image_shape": [480, 640],
        "normalization": "imagenet",
        "num_copies": 2,
        "num_frames": 1,

        "sampling": {
            "NYUv2Depth": 1.0
        },
        "train_datasets": [
            "NYUv2Depth"
        ],
        "val_datasets": [
            "NYUv2Depth"
        ],
        "val_display_names": {
            "NYUv2Depth": "NYUv2"
        },

        "data_root": "/home/xzy/datasets",
        "val_no_gt_camera": false,
        "crop": "garg",

        "augmentations": {
            "random_scale": 2.0,
            "random_jitter": 0.4,
            "jitter_p": 0.8,
            "random_blur": 2.0,
            "blur_p": 0.2,
            "random_gamma": 0.2,
            "gamma_p": 0.8,
            "grayscale_p": 0.2,
            "flip_p": 0.5,
            "test_context": 1.0,
            "shape_constraints": {
                "ratio_bounds": [0.5, 2.5],
                "pixels_max": 600000,
                "pixels_min": 200000,
                "height_min": 15,
                "width_min": 15,
                "shape_mult": 14,
                "sample": true
            }
        }
    },

    "model": {
        "name": "UniDepthV2",
        "num_heads": 8,
        "expansion": 4,
        "layer_scale": 1.0,

        "pixel_decoder": {
            "name": "Decoder",
            "hidden_dim": 512,
            "dropout": 0.0,
            "depths": [2, 2, 2],
            "out_dim": 64,
            "kernel_size": 3
        },

        "pixel_encoder": {
            "lr": 2e-6,
            "wd": 0.1,
            "name": "dinov2_vitl14",
            "frozen_stages": 0,
            "num_register_tokens": 0,
            "use_norm": true,
            "freeze_norm": true,
            "pretrained": null,
            "stacking_fn": "last",
            "output_idx": [6, 12, 18, 24]
        }
    }
}
`
